lib/gpu_memory_service/integrations/vllm/model_loader.py

```
@@ -346,8 +363,112 @@ def _load_write_mode(
+    with set_default_torch_dtype(model_config.dtype):
+        with device:
+            model = initialize_model(vllm_config=vllm_config, model_config=model_config)
+        # Runs on real tensors, so post-load hooks that build derived tensors
+        # (e.g. MLA projections, MoE kernels) execute normally rather than being
+        # skipped/swallowed as on the meta path. Values are garbage and are
+        # overwritten by materialize; only the resulting structure matters. A
+        # garbage-value-sensitive hook is unexpected but should not abort
+        # bring-up — materialize + the unbacked-parameter check cover the rest.
+        try:
+            process_weights_after_loading(model, model_config, device)
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)